packages/components/src/styles/components/app-header.scss

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APP-HEADER
+//
+
+@use "../mixins/focus-ring" as *;
+
+// TODO: Copy & pasted, clean up
+@mixin hds-side-nav-icon-button($add-visible-border: false) {
+  color: var(--token-side-nav-color-foreground-strong);
+  background-color: transparent;
+  border: 1px solid transparent;
+  border-radius: var(--token-side-nav-body-list-item-border-radius);
+  cursor: pointer;
+
+  @if ($add-visible-border) {
+    border-color: var(--token-color-palette-neutral-500);
+  }
+
+  &:focus,
+  &.mock-focus {
```

Comment:
I think this is getting overridden somewhere with the default styles
<img width="391" alt="Screenshot 2024-06-14 at 1 36 37 PM" src="https://github.com/hashicorp/design-system/assets/2200899/136e4739-db80-434c-b6f8-44516af4759d">

Reply:
Thanks, I haven't worked on "real" styles for the dropdown and button content yet (was waiting for visual designs to be more nailed down before over-focusing on styling details so I mainly focused on the AppHeader and the slots where things get dropped in to start) so haven't looked at focus styles or anything like that. Those are part of what I want to focus on next which is part of why I brought up the questions about icon sizing and how best to handle the dark "theme".

Reply:
Just overhauled all the interactive children styles (Buttons & Dropdowns). Added examples to Showcase. Now ready for visual design reviews. @jorytindall

Reply:
Still need to make updates to Dropdown icons, etc. next BTW.